File: Visual Studio Windows Forms/C#/Spring Break Travel Specials/Spring Break Travel Specials/Form1.cs
/*
 * Programmer:      Corey Jenkins
 * Date:            February 17, 2021 (Updated February 8, 2023)
 * Program Name:    Spring break Travel Specials
 * Purpose:         To create a program in which you can select which destination you want to go.
 *                  Once selected, it cannot be changed again. A message will appear once it has been selected.
*/

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Spring_Break_Travel_Specials
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }

        private void btnMyrtleBeach_Click(object sender, EventArgs e)
        {
            picMyrtleBeach.Visible = true;
            picVeniceBeach.Visible = false;
            picKeyWest.Visible = false;

        }

        private void btnKeyWest_Click(object sender, EventArgs e)
        {
            picKeyWest.Visible = true;
            picVeniceBeach.Visible = false;
            picMyrtleBeach.Visible = false;

        }

        private void btnVeniceBeach_Click(object sender, EventArgs e)
        {
            picVeniceBeach.Visible = true;
            picKeyWest.Visible = false;
            picMyrtleBeach.Visible = false;
        }

        private void btnPurchase_Click(object sender, EventArgs e)
        {
            btnKeyWest.Enabled = false;
            btnVeniceBeach.Enabled = false;
            btnMyrtleBeach.Enabled = false;
            btnPurchase.Enabled = false;
            lblEnjoy.Visible = true;

        }

        private void btnCloseWindow_Click(object sender, EventArgs e)
        {
            Close();
        }

        private void aboutToolStripMenuItem_Click(object sender, EventArgs e)
        {
            string message = "This program was created by Corey Jenkins.";
            string caption = "About this program";
            MessageBox.Show(message, caption);
        }
    }
}

File: Visual Studio Windows Forms/Visual Basic/Spring Break Travel Specials/Spring Break Travel Specials/Form1.vb
' Program name: Spring Break Travel Specials
' Developer: Corey Jenkins
' Date: September 13, 2017
' Purpose: This application will be used to select between the locations of Myrtle Beach, Key West,
' Venice Beach. The user can only choose one.
Public Class FrmSpringBreakTravelSpecial
    Private Sub btnMyrtleBeach_Click(sender As Object, e As EventArgs) Handles btnMyrtleBeach.Click
        ' This button will enabled the picture of Myrtle Beach and Key West, and Venice Beach will not show.


        picMyrtleBeach.Visible = True
        picKeyWest.Visible = False
        picVeniceBeach.Visible = False
        btnPurchase.Enabled = True
    End Sub

    Private Sub btnKeyWest_Click(sender As Object, e As EventArgs) Handles btnKeyWest.Click
        ' This button will enabled the picture of Key West while Myrtle Beach and Venice Beach will not appear.
        ' The Purchase button is also enabled.

        picKeyWest.Visible = True
        picMyrtleBeach.Visible = False
        picVeniceBeach.Visible = False
        btnPurchase.Enabled = True
    End Sub

    Private Sub btnVeniceBeach_Click(sender As Object, e As EventArgs) Handles btnVeniceBeach.Click
        ' This button will enable the picture of Venice Beach and Myrtle Beach and Key West pictures will not appear.
        ' The Purchase Button will also be enabled.

        picVeniceBeach.Visible = True
        picMyrtleBeach.Visible = False
        picKeyWest.Visible = False
        btnPurchase.Enabled = True
    End Sub

    Private Sub btnPurchase_Click(sender As Object, e As EventArgs) Handles btnPurchase.Click
        ' This is for the Purchase button. When enabled none of the buttons can be clicked.
        ' The only option from this point will be the Exit Button.
        lblEnjoy.Visible = True
        btnPurchase.Enabled = False
        btnMyrtleBeach.Enabled = False
        btnKeyWest.Enabled = False
        btnVeniceBeach.Enabled = False
        btnCloseWindow.Enabled = True
    End Sub

    Private Sub btnCloseWindow_Click(sender As Object, e As EventArgs) Handles btnCloseWindow.Click
        ' This will close the window once a location has been purchased.
        Close()
    End Sub
End Class